Tools/vagrant/initvagrant.sh

```diff
@@ -1,24 +1,36 @@
 #!/bin/bash
+echo "---------- initvagrant.sh start ----------"
 
 # this script is run by the root user in the virtual machine
 
 set -e
 set -x
 
+who=$(whoami)
 echo "Initial setup of SITL-vagrant instance."
+if [ $who != 'root' ]; then
+    echo "SORRY, MUST RUN THIS SCRIPT AS ROOT, GIVING UP"
+    exit 1
+fi
 
 VAGRANT_USER=ubuntu
 if [ -e /home/vagrant ]; then
     # prefer vagrant user
     VAGRANT_USER=vagrant
 fi
+echo USING VAGRANT_USER:$VAGRANT_USER
```

Comment:
Isn't a lot of this coming out via `set -x` anyway?  Next thing people will want to remove `-x` again :-/

Reply:
In my setup at least, an explicit 'echo' actually puts the the output in green text, while the rest of the verbose text goes past in red, so the echo stands out.     I had this especially stand out like this, as the choice to use /home/vagrant over /home/ubuntu is important for a bunch of scripts later on ( such as where jsbsim actually gets installed ).